pyaim/aimcp.py

```diff
@@ -0,0 +1,65 @@
+import os
+import time
+from subprocess import PIPE, Popen
+from sys import platform
+
+
+class CLIPasswordSDK(object):
+
+    def __init__(self, cli_path):
+
+        self._cli_path = cli_path
+
+        # Check for Linux OS
+        if platform == 'linux' or platform == 'linux2':
+            self.sep = '-'
+            if not os.path.isfile(self._cli_path):
+                raise Exception('CLIPasswordSDK not found', 'Please make sure you provide a proper path to CLIPasswordSDK.')
+        # Check for Windows OS
+        elif platform == 'win32':
+            self.sep = '/'
+            if not os.path.isfile(self._cli_path):
+                raise Exception('CLIPasswordSDK.exe not found', 'Please make sure you provide a proper path to CLIPasswordSDK.exe.')
+        # Check for MacOS
+        elif platform == 'darwin':
+            raise Exception('Unsupported OS', 'Support only exists for Linux & Windows currently.')
+        # Unknown platform returned
+        else:
+            raise Exception('Cannot detect OS', 'Your platform is unrecognizable. Please use Linux or Windows.')
+
+
+    def GetPassword(self, appid=None, safe=None, obj=None):
+        # Check appid is declared
+        if appid is None:
+            raise Exception('No Application ID', 'Please declare a valid Application ID.')
+        # Check safe is declared
+        elif safe is None:
+            raise Exception('No Safe Name', 'Please declare a valid Safe name.')
+        # Check object name is declared
+        elif obj is None:
+            raise Exception('No Object Name', 'Please declare a valid Object name.')
+        else:
+            try:
+                response, err = Popen(
+                    [
+                        '{}'.format(self._cli_path),
```

Comment:
if you're using a value as-is, you don't need this at all, just set the value directly: `self._cli_path`